mllib/src/main/scala/org/apache/spark/ml/classification/RandomForestClassifier.scala

```diff
@@ -222,16 +222,10 @@ final class RandomForestClassificationModel private[ml] (
   /**
    * Estimate of the importance of each feature.
    *
-   * This generalizes the idea of "Gini" importance to other losses,
-   * following the explanation of Gini importance from "Random Forests" documentation
-   * by Leo Breiman and Adele Cutler, and following the implementation from scikit-learn.
+   * Each feature's importance is the average of its importance across all trees in
```

Comment:
This doc explaining how the feature importances were computed was copied in the code ~6 times previously. I changed it to reduce the redundancy. Now that DTs have a public scala doc explaining how single tree importance is computed, I thought it would be good to leave a reference to that doc here instead.